src/java/picard/analysis/directed/TargetMetricsCollector.java

```diff
@@ -538,14 +538,15 @@ public void acceptRecord(final SAMRecord record) {
                         boolean isOnTarget = false;
                         for (final Interval target : targets) {
                             if (refPos >= target.getStart() && refPos <= target.getEnd()) {
-                                isOnTarget = true;
                                 ++this.metrics.ON_TARGET_BASES;
                                 if (mappedInPair) ++this.metrics.ON_TARGET_FROM_PAIR_BASES;
 
                                 final int targetOffset = refPos - target.getStart();
                                 final Coverage coverage = this.coverageByTarget.get(target);
                                 coverage.addBase(targetOffset);
                                 baseQHistogramArray[baseQualities[offset]]++;
+                                if (!isOnTarget) coverage.addReadCount();
+                                isOnTarget = true;
```

Comment:
Looking at the change here, this will _only_ increment the read count for the _first_ target that a read maps over, which seems a little odd.  E.g. if you have two targets close together, separated by 20 bases, both at ~100X with most reads straddling both you could end up with the second target having ~100X coverage but very low read count.